vaadin-form-layout-flow-parent/vaadin-form-layout-flow/src/main/java/com/vaadin/flow/component/formlayout/FormLayout.java

```diff
@@ -449,4 +449,16 @@ public FormItem addFormItem(Component field, Component label) {
         add(formItem);
         return formItem;
     }
+
+    /**
+     * Set the width of the label when aside.
+     * 
+     * @param width
+     *          the value and CSS unit
+     * @see <a href=
+     *      "https://vaadin.com/docs/latest/components/form-layout#label-position">Label position</a>
+     */
+    public void setLabelWidth(String width) {
```

Comment:
Should we go with the naming suggestion by Rolf? Apart from the JavaDoc it's not obvious otherwise that this only applies to specific labels / with a specific configuration. Personally I'm also fine with the current naming. @DiegoCardoso Do you have an opinion on this?

Reply:
I intentionally went with `setLabelWidth` as I don't think `setLabelAsideWidth` is any better. I don't think `setLabelColumnWidth` is too good either as FormLayout is based around columns, and this, the way I see, has nothing to do with said columns. But you're right that the method name does not help the user realize it only applies to side-positioned labels.

Reply:
I think the name is fine and the JavaDoc is there to help. 

Also, I think a getter should be added too.